lib/internal/Magento/Framework/Webapi/Rest/Request.php

```diff
@@ -11,6 +11,13 @@
 use Magento\Framework\Api\SimpleDataObjectConverter;
 use Magento\Framework\Phrase;
 
+/**
+ * Class Request
+ *
+ * @api
+ *
+ * @package Magento\Framework\Webapi\Rest
```

Comment:
Hi @engcom-Hotel, I don't think there is a need for `package` annotation.

Reply:
Has been removed.